client/src/components/HistoryModal.tsx

```diff
@@ -26,10 +29,34 @@ const HistoryModal: React.FC<HistoryModalProps> = ({
           {item?.summaryTitle} - {item?.date}
         </h2>
         <p>{item?.summary}</p>
-        <div className="flex justify-end mt-4">
-          <Button onClick={onClose} variant="active">
-            Close
-          </Button>
+        <div className="flex justify-evenly mt-12">
+          <div className="flex mt-4">
+            <input
+              type="text"
+              style={{ width: "20rem", padding: "1rem" }}
+              value={emailBody}
+              onChange={(e) => setEmailBody(e.target.value)}
+              placeholder="Enter your email"
+            />
+          </div>
+          <div className="flex mt-4 ">
```

Comment:
After doing a live test, you should disable the `Email Summary` btn until the user has entered smth into the email textbox